.github/workflows/ci.yml

```
@@ -0,0 +1,95 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - release-[0-9].*
```

Why not the value from the template? (I don't know how many years it would take but this wouldn't work if we got to version 10.something.)
```suggestion
      - release-*
```